ipn/ipnlocal/local.go

```diff
@@ -566,10 +566,18 @@ func (b *LocalBackend) findExitNodeIDLocked(nm *netmap.NetworkMap) (prefsChanged
 func (b *LocalBackend) setWgengineStatus(s *wgengine.Status, err error) {
 	if err != nil {
 		b.logf("wgengine status error: %v", err)
+
+		b.statusLock.Lock()
```

Comment:
Move some of the commit message (which ~nobody will ever see again) to a comment here in the code to explain what's happening.

Also, no need to Lock/Unlock: https://golang.org/pkg/sync/#Cond.Broadcast says:

> It is allowed but not required for the caller to hold c.L during the call.

Reply:
I guess you merged before seeing this comment.